src/instructlab/lab.py

```diff
@@ -968,39 +951,58 @@ def convert(self, value, param, ctx) -> "torch.device":
     ),
 )
 @click.option(
-    "--model-name",
-    default="instructlab/merlinite-7b-lab",
+    "--dtype",
+    type=click.Choice(["bf16", "fp16", "fp32", "auto"]),
+    show_default=True,
+    default="auto",
+    help=("Full half precision training. BF16 is more precise than FP16."),
+)
+@click.option(
+    "--backend",
+    type=click.Choice(["pytorch", "mlx"]),
     show_default=True,
-    help="model name to use in training",
+    default="pytorch",
+    help=("backend to use for training. Options are mlx or pytorch"),
+)
+@click.option(
+    "--train-style",
+    type=click.Choice(["quick", "full"]),
+    show_default=True,
+    default="quick",
```

Comment:
Should we default to `full` instead of  `quick`? In case users aren't aware of this flag